docs/docs/install.md

```diff
@@ -11,6 +12,25 @@ pip install -e .
 
 Installing `delfi` as described above will automatically take care of the requirements. In total, this installation will typically take less than a minute.
 
+## GPU support from a fresh Anaconda installation
+To install delfi and all prerequisites totally from scratch, follow these steps for Ubuntu 19.10 (last updated Jan 20, 2020).
+
+* Install the correct CUDA driver for your GPU, if not already installed
+* Install ```libopenblas-dev``` if you haven't done so already (see below)
+* Install CUDA and cuDNN if not already installed
+* Download, install and activate Anaconda for python 3 ([instructions](https://docs.anaconda.com/anaconda/install/linux/))
+* Close any terminal windows and make a new one, and run these commands:
+```
+conda install ipython numpy scipy pytest matplotlib mkl mkl-service
+pip install theano
+conda install pygpu
+echo -e "\n[blas]\nldflags = -lopenblas\n" >> ~/.theanorc
+```
+* Open a terminal, change to the base directory of the delfi repository, and run these commands:
+```
+pip install requirements.txt
```

Comment:
`pip install -r requirements.txt`? 

Looks good to me otherwise, feel free to merge

Reply:
right, thanks